runtime/ftplugin/hlsplaylist.vim

```diff
@@ -0,0 +1,39 @@
+" Vim filetype plugin
+" Language:     M3U Playlist
+" Maintainer:	AvidSeeker <avidseeker7@protonmail.com>
+" Last Change:  2024 Jul 07
+"
+
+if exists("b:did_ftplugin")
+  finish
+endif
+let g:did_ftplugin = 1
+
+setlocal commentstring=#%s
+
+if exists('b:undo_ftplugin')
+  let b:undo_ftplugin .= "|setl commentstring<"
+endif
```

Comment:
```
if exists('b:undo_ftplugin')
  let b:undo_ftplugin .= "|setl commentstring<"
endif
```

Instead of the `if/endif` simply use:

```vim
 let b:undo_ftplugin = "setl commentstring<"
```